Solution1/ClassLibrary1/CustomButton.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Drawing;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Windows.Forms;
+
+namespace ClassLibrary1
+{
+    class CustomButton : IControlItem
+    {
+        public string Text { get; set; }
+
+        public Color BackgroundColor = Color.White;
+
+        public EventHandler OnMouseClick { get; set; }
+
+        public void OnPaint(Rectangle area, Graphics graphics)
+        {
+            graphics.FillRectangle(
```

Comment:
please check this [doc](https://learn.microsoft.com/ru-ru/dotnet/api/system.idisposable?view=net-7.0)